integration/prefetch-test.ts

```diff
@@ -185,7 +185,29 @@ test.describe("prefetch=intent (hover)", () => {
       "#nav link[rel='modulepreload'][href^='/build/routes/without-loader-']",
       { state: "attached" }
     );
-    expect(await page.locator("#nav link").count()).toBe(3);
+    expect(await page.locator("#nav link").count()).toBe(1);
```

Comment:
@ryanflorence Now that we `setShouldPrefetch(false)` in `cancelIntent` these tests are invalid since the link tags remove - just confirming that's expected 👍